app/forms/mappings/Formatters.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.mappings
+
+import cats.implicits._
+import play.api.data.FormError
+import play.api.data.format.Formatter
+
+import scala.util.chaining._
+import scala.util.control.Exception.nonFatalCatch
+
+trait Formatters {
```

Comment:
can we make it object and don't extend it but import it - it would be losely coupled then. Inheritenace/Mixin is too strong - we'll follow **the principle of the least power**